src/test/java/net/sf/jabref/logic/exporter/ModsExportFormatTestFiles.java

```diff
@@ -65,7 +62,7 @@ public void setUp() throws Exception {
         charset = StandardCharsets.UTF_8;
         modsExportFormat = new ModsExportFormat();
         tempFile = testFolder.newFile();
-        testImporter = new BibtexImporter(ImportFormatPreferences.fromPreferences(Globals.prefs));
+        testImporter = new BibtexImporter(Globals.prefs.getImportFormatPreferences());
     }
 
     @Test
```

Comment:
I think @Siedlerchr meant StandardCharsets.UTF_8, which it looks like you are using, at least assigning it to charset.